airbnb_project/database/airbnb_db_creator.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS listing (
+    airbnb_listing_id VARCHAR(20) PRIMARY KEY,
+    baths VARCHAR(10),
+    beds VARCHAR(10),
```

Comment:
Ok a couple of thins here:
- maybe confirm with @immangat  but I think `baths`, `beds` and `person_capacity` should be `INTEGER`
- lat/lng could probably be `DECIMAL(8,6), DECIMAL(9,6)`, respectively
- `title` can be fairly long too so I'd suggest 200